e2e/integration/frontity-01/script.spec.js

```diff
@@ -17,13 +16,11 @@ describe("Script", () => {
   });
 
   it("should access code from the external script", () => {
-    let isMoment;
     cy.window()
-      .then((win) => {
-        isMoment = win.moment()._isAMomentObject;
-      })
-      .then(() => {
-        expect(isMoment).toBe(true);
+      .its("moment")
+      .should("exist")
+      .then((moment) => {
+        expect(moment()._isAMomentObject).toBe(true);
```

Comment:
I think this could be rewritten like this, right? Just to follow the Cypress way of do things and avoid dealing with jest eslint rules. :shrug: 

```ts
cy.window()
  .its("moment")
  .should("exist")
  .then((moment) => moment()._isAMomentObject)
  .should("be.true"); // or .should("equal", true)
```

Reply:
That's a good catch @DAreRodz. Like the way it integrates.